src/wast-lexer.cc

```diff
@@ -64,6 +64,9 @@ Token WastLexer::GetToken(WastParser* parser) {
             continue;
           }
           return BareToken(TokenType::Eof);
+        } else if (MatchString("(@")) {
+          ReadReservedChars();
+          return TextToken(TokenType::LparAnn);
```

Comment:
Should be TextToken(..., 2), to skip the "(@"

Reply:
done in https://github.com/WebAssembly/wabt/pull/1076/commits/03bdf8e5d7c91f10062ce767e09ce34a575bdcff